cmd/application.go

```diff
@@ -132,21 +133,40 @@ var applicationListCmd = &cobra.Command{
 	Use:   "list",
 	Short: "Lists all the applications",
 	Long:  "Lists all the applications",
-	Example: `  # List all applications
+	Example: `  # List all applications in current project
   odo app list
+
+  # List all applications in the specified project
+  odo app list --project myproject
 	`,
 	Args: cobra.ExactArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
 		client := getOcClient()
-		apps, err := application.List(client)
-		checkError(err, "")
-		fmt.Printf("ACTIVE   NAME\n")
-		for _, app := range apps {
-			activeMark := " "
-			if app.Active {
-				activeMark = "*"
+		if projectName == "" {
+			apps, err := application.List(client)
+			checkError(err, "")
+			fmt.Printf("ACTIVE   NAME\n")
+			for _, app := range apps {
+				activeMark := " "
+				if app.Active {
+					activeMark = "*"
+				}
+				fmt.Printf("  %s      %s\n", activeMark, app.Name)
```

Comment:
Since you are anyway modifying this section, it would be nice if you can change this to use tabwriter as in https://github.com/redhat-developer/odo/blob/master/cmd/catalog.go#L54-L59